src/jbuild

```diff
@@ -4,13 +4,19 @@
  ((name lambda_term)
   (public_name lambda-term)
   (wrapped false)
-  (libraries (lwt lwt.unix lwt_react zed))
+  (libraries
+   (lwt lwt_react zed
+    (select lTerm_sequence.ml from
+     (lwt.unix -> lTerm_sequence_lwt3.ml)
+     (lwt-unix -> lTerm_sequence_lwt4.ml))
```

Comment:
I think ```lwt 4.0.0``` still defines ```lwt.unix``` findlib package as ```lwt.unix``` and not ```lwt-unix```. See https://github.com/ocsigen/lwt/blob/master/src/unix/jbuild#L56. Isn't the dune ```(library (public_name)``` field the findlib package name?

Incidentally, I tried ```ocamlfind query lwt-unix``` and it returns with ```ocamlfind: Package `lwt-unix' not found```. and when I do ```ocamlfind query lwt.unix``` it returns the correct correct path.

cc @aantron. Were you looking to rename ```lwt.unix``` to ```lwt-unix``` in ```lwt 4.0.0```?

Reply:
Indeed, `lwt.unix` is correct, and we haven't renamed it.

Reply:
Ah yes, ok so we'll need a different check. Unfortunately, we might not be able to do with a select form anymore, which will complicate things.